app/presenters/workflow_process_presenter.rb

```diff
@@ -23,25 +17,40 @@ def elapsed
     format('%.3f', process_status.elapsed.to_f)
   end
 
-  def reset_button
-    return unless new_status
-
-    # workflow update requires id, workflow, process, and status parameters
-    form_tag item_workflow_path(pid, workflow_name), method: 'put' do
-      hidden_field_tag('process', name) +
-        hidden_field_tag('status', new_status) +
-        button_tag('Set to ' + new_status, type: 'submit')
+  def status_action
+    case status
+    when 'error'
+      error_choices
+    when 'waiting'
+      completed_button('completed')
```

Comment:
could this also be 'skipped'?  I'm not clear on the difference ...

Reply:
I'm not sure I understand.  This is the existing functionality for "waiting".  In https://github.com/sul-dlss/argo/issues/1346#issuecomment-521484228 we agreed the ticket was only affecting rows with the "error" status.